runtime/gc_vlhgc/SchedulingDelegate.cpp

```diff
@@ -1355,37 +1355,25 @@ MM_SchedulingDelegate::calculateEdenSize(MM_EnvironmentVLHGC *env)
 	 * If heap is fully expanded (or close to) make sure that there are enough free regions to satisfy given eden size change
 	 */
 	intptr_t maxEdenChange = 0;
-	uintptr_t maxEdenRegionCount = _extensions->getHeap()->getHeapRegionManager()->getTableRegionCount();
-	bool edenIsVerySmall = (_edenRegionCount * 64) < maxEdenRegionCount;
+	/* Total heap needs to be aware that by changing eden size, the amount of survivor space might also need to change */
```

Comment:
This part of comment 'Total heap needs to be aware that by changing eden size,'  can be removed
We state the same in the next comment.
This comment is just about adjusting for survivor. Could be as simple as 'Proportionally adjust survivor area.'